hosthome.py

```diff
@@ -122,7 +122,18 @@ def get_element_by_id(self, collection_name, id):
 
         self._log('get_collection', 'item = {}'.format(item))
         return item
+    ###########################################################attempt
```

Comment:
Previously discussed with back end team: we are leaving this hard-coded value temporarily to test our authentication+registration flow. This will be made dynamic